framework/src/main/java/org/checkerframework/common/accumulation/AccumulationValue.java

```diff
@@ -0,0 +1,139 @@
+package org.checkerframework.common.accumulation;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.TransferResult;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.framework.flow.CFAbstractAnalysis;
+import org.checkerframework.framework.flow.CFAbstractValue;
+import org.checkerframework.javacutil.AnnotationMirrorSet;
+
+/**
+ * AccumulationValue holds additional information about accumulated facts ("values", not to be
+ * confused with "Value" in the name of this class) that cannot be stored in the accumulation type,
+ * because they are not a refinement of that type. This situation occurs for type variables and
+ * wildcards, for which calling {@link AccumulationTransfer#accumulate(Node, TransferResult,
+ * String...)} would otherwise have no effect (since the types are invariant: Accumulator(a) T is
```

Comment:
This isn't true. `Accumulator(a) T` is a supertype of `Accumulator(a,b) T` but `T` is not a supertype of `Accumulator(a) T` unless both bounds of `T` are supertypes of of `Accumulator(a)`.